script/Makefile

```diff
@@ -456,6 +456,18 @@ images-all:
 	make IMAGE_ARCH=arm64 images
 	make IMAGE_ARCH=amd64 images
 
+images-push:
+	docker push $(CUSTOM_IMAGE):$(CUSTOM_VERSION)-amd64
+	docker push $(CUSTOM_IMAGE):$(CUSTOM_VERSION)
+	@if docker inspect $(CUSTOM_IMAGE):$(CUSTOM_VERSION)-arm64 &> /dev/null; then \
+		echo "Image $(CUSTOM_IMAGE):$(CUSTOM_VERSION)-arm64 exists, building the multiarch manifest"; \
+		docker push $(CUSTOM_IMAGE):$(CUSTOM_VERSION)-arm64; \
+		docker manifest create $(CUSTOM_IMAGE):$(CUSTOM_VERSION) --amend $(CUSTOM_IMAGE):$(CUSTOM_VERSION)-amd64 --amend $(CUSTOM_IMAGE):$(CUSTOM_VERSION)-arm64; \
+		docker manifest push --purge $(CUSTOM_IMAGE):$(CUSTOM_VERSION); \
+	else \
+		echo "Image $(CUSTOM_IMAGE):$(CUSTOM_VERSION)-arm64 does not exist"; \
+	fi
+
 images-push-staging:
```

Comment:
We need to remove this, then.

Reply:
No, this is intentional.

The general idea is that `make images` (ignore the confusing plural) builds the image for your current platform. Hence, there is no guarantee that *-arm64 will be available.

Only if you do `make images-all` will all the images be available - see the GHA for this

`make images-push` should fail when *-amd64 is not available. In that way it is guaranteed, that the default version (without arch suffix) always points to: either the -amd64 variant or a manifest list that contains the -amd64 variant.